manifest.json

```diff
@@ -15,24 +15,24 @@
     "postreleasy": "vtex publish --verbose"
   },
   "dependencies": {
+    "vtex.store": "1.x",
     "vtex.store-graphql": "2.x",
-    "vtex.shelf": "1.x",
+    "vtex.store-header": "2.x",
+    "vtex.store-footer": "2.x",
+    "vtex.store-components": "3.x",
+    "vtex.category-menu": "2.x",
     "vtex.menu": "2.x",
+    "vtex.login": "2.x",
     "vtex.minicart": "2.x",
-    "vtex.my-account": "0.x",
-    "vtex.store-components": "3.x",
-    "vtex.search-result": "3.x",
-    "vtex.product-summary": "2.x",
+    "vtex.telemarketing": "2.x",
+    "vtex.shelf": "1.x",
     "vtex.carousel": "2.x",
-    "vtex.product-details": "1.x",
-    "vtex.category-menu": "2.x",
+    "vtex.search-result": "3.x",
     "vtex.breadcrumb": "1.x",
-    "vtex.login": "2.x",
-    "vtex.styleguide": "8.x",
-    "vtex.store": "1.x",
+    "vtex.product-details": "1.x",
+    "vtex.product-summary": "2.x",
     "vtex.product-kit": "1.x",
-    "vtex.telemarketing": "2.x",
-    "vtex.store-header": "2.x",
-    "vtex.store-footer": "2.x"
+    "vtex.my-account": "0.x",
```

Comment:
Are all these dependencies necessary? For example, `vtex.my-account` and `vtex.product-summary` don't seem to be used in `blocks.json`.